apps/codeforafrica/src/payload/fields/dateTime.js

```diff
@@ -0,0 +1,17 @@
+import { deepmerge } from "@mui/utils";
```

Comment:
Can't we reuse `publishedOn` (or at least create one reusable `dateTime` field)?